### ISSUE TITLE
colorscheme question

### ISSUE THREAD

**Issue by ywpkwon** [yuki-yano/fzf-preview.vim#90]
Based on #24 , I set a custom `g:fzf_preview_grep_preview_cmd` and `$BAT_THEME='Monokai Extended'` and have a correct color scheme in preview as in below (the right panels in preview).
#### FzfPreviewProjectGrep
`:FzfPreviewProjectGrep -add-fzf-arg=--nth=3 .`
![image](https://user-images.githubusercontent.com/18429284/75960443-ecc87980-5e74-11ea-868e-1fe61fa0592b.png)

#### FzfPreviewProjectFiles
`:FzfPreviewProjectFiles`
![image](https://user-images.githubusercontent.com/18429284/75960397-d6222280-5e74-11ea-87aa-ac3a71e3e75f.png)

However, where do the colors of the **left** panel in `:FzfPreviewLines` come from (see below)? How can I change those colors?
#### FzfPreviewLines
`:FzfPreviewLines`
![image](https://user-images.githubusercontent.com/18429284/75960694-5b0d3c00-5e75-11ea-895e-4d0c4c68a285.png)

**Comment by yuki-yano** [comment]
Now it's hardcoded into the preview script.
https://github.com/yuki-ycino/fzf-preview.vim/blob/master/bin/preview_fzf_grep#L23

You can now work around this by copying the preview script, placing it in an executable location, and changing the theme.
For example, `let g:fzf_preview_grep_preview_cmd = 'my_preview_fzf_grep'`

I am thinking about setting `$FZF_PREVIEW_PREVIEW_BAT_THEME` to change the theme.
`let $FZF_PREVIEW_PREVIEW_BAT_THEME = 'Monokai Extended'`
What do you think?

**Comment by ywpkwon** [comment]
@yuki-ycino Thanks for the explanation. I think I did exactly what you said. So, I was able to have the correct color in `:FzfPreviewProjectGrep` and `:FzfPreviewLines` (right panel) in my screenshots. (As you see all right panels have `monokai` in my three screenshots). 
I would like to ask what about the **left** panel. In `FzfPreviewLines`, the left panel is also showing codes and it has some kind of syntax highlighting. I was wondering where this comes from. Am I missing something?
[p.s.] Btw, I think `let $FZF_PREVIEW_PREVIEW_BAT_THEME = 'Monokai Extended'` solution looks good.

**Comment by yuki-yano** [comment]
Sorry, I misread it.
GetLines command is defined from
https://github.com/yuki-ycino/fzf-preview.vim/blob/master/autoload/fzf_preview/command.vim#L46

This command must be configurable.
I will improve it.

**Comment by ywpkwon** [comment]
Thanks a lot for the quick response. I'll try. Back to your earlier answer, what I was doing was just using `$BAT_THEME` and removing `--theme` everywhere. I got this hint from #24.
```
    let $BAT_THEME='Monokai Extended'
    let g:fzf_preview_command = 'bat --color=always --style=grid {-1}'
    let g:fzf_preview_grep_preview_cmd = '~/.config/nvim/preview_fzf_grep'
```
I guess `$FZF_PREVIEW_PREVIEW_BAT_THEME` would be more clearer. :)

**Comment by yuki-yano** [comment]
Fixed #93
